dashboard/api.py

```diff
@@ -93,6 +93,13 @@ def __init__(self, status, course_run=None, enrollment_for_course=None):
         self.course_run = course_run
         self.enrollment_for_course = enrollment_for_course
 
+    def __repr__(self):
+        return "<CourseRunUserStatus for course {course} status {status} at {address}>".format(
+            status=self.status,
+            course=self.course_run.title if self.course_run is not None else '"None"',
```

Comment:
It was to have the string with the quotes in the `repr` string, this is a weird case and I wanted to make it more visible.